src/context/mod.rs

```diff
@@ -858,6 +841,13 @@ impl<'ctx, CTX: private::SealedContext<'ctx> + private::SealedCanReadState> Cont
 /// # }
 /// ```
 ///
+/// ## Command-line introspection
+/// You can inspect and edit the K/V state stored in Restate via `psql` and the CLI.
+/// Have a look at the [introspection docs](https://docs.restate.dev//operate/introspection#inspecting-application-state) for more information.
+///
+/// ### Serializing state
+/// You can store any type of value that that implements the `serde::Serialize` and `serde::Deserialize` traits ([see serialization docs][crate::serde]).
+///
```

Comment:
It is very confusing here to use `serde::Serialize`, because it will lead people to believe this is `serde.rs` traits. I would either shorten the sentence to something simple like _Look at serialization docs for more details_ or just blindly remove it, it's in the docs of the method anyway.